### ISSUE TITLE
About logits for ImageNet

### ISSUE THREAD

**Issue by tomchen-ctj** [CHENGY12/PLOT#2]
Hi, 

Thank you for the solid work and code releases!
It seems that for ImageNet training, an additional logits were calculated as ['pooled logits'](https://github.com/CHENGY12/PLOT/blob/main/plot-coop/trainers/plot.py#L278). May I ask about the actual benefits of this operation? (Sorry if I missed that description in your paper)

Thanks!

**Comment by CHENGY12** [comment]
Thanks very much for your attention to our work.  'pooled logits' denotes the original Euclidean distance between the average of visual features and prompted text features.  By fusing this global distance and local matching distance with optimal transport, PLOT can achieve a better balance between  features in different granularities. Please let me know if you have any questions.

**Comment by tomchen-ctj** [comment]
I see, thanks for your reply!
